### PR TITLE
Cart 52 visitor counter

### PR DESCRIPTION
**Pull request by vetalcore**

### CODE REVIEW

File: dev/publicDev/index.jade [hunk 1]
```diff
@@ -7,8 +7,6 @@ html(lang='en-US', ng-app='app')
     link(rel='stylesheet', href='app.css')
   body(ng-controller="mainCtrl as vm")
     .main-view(ng-view)
-      footer
```
Comment (dmitrij-borchuk) [inline dev/publicDev/index.jade:11]: Part of the design

File: dev/publicDev/scripts/mainCtrl.js [hunk 2]
```diff
@@ -2,10 +2,21 @@
   "use strict";
   var app = angular.module('app');
 
-  ctrl.$inject = ['$http'];
-  function ctrl ($http) {
+  function dataService($http) {
+    this.getVisitorsCount = function() {
+      return $http.get('api/settings/visitorsCounter');
```
Comment (dmitrij-borchuk) [inline dev/publicDev/scripts/mainCtrl.js]: Rename to 'api/visitorsCounter'

File: dev/routing.js [hunk 3]
```diff
@@ -7,14 +7,17 @@ const DAL = require('./dal/dal.js');
 const Joi = require('joi');
 const files = require('./routing/files.js');
 const Boom = require('boom');
+const Utils = require('./services/utils.js');
+
+let count = 0;
```
Comment (dmitrij-borchuk) [inline dev/routing.js]: What for?

File: dev/routing.js [hunk 4]
```diff
@@ -7,14 +7,17 @@ const DAL = require('./dal/dal.js');
...
 
 module.exports.init = function (server) {
   server.route({
     method: 'GET',
     path: '/',
     handler: function (request, reply) {
-      reply.file( path.resolve(__dirname, './public/index.html') );
-    }
+        reply.file( path.resolve(__dirname, './public/index.html') );
```
Comment (dmitrij-borchuk) [inline dev/routing.js]: What for?

File: dev/routing/settings.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const path = require('path');
+const DAL = require('../dal/dal.js');
+
+module.exports = function (server) {
+  server.route({
```
Comment (dmitrij-borchuk) [inline dev/routing/settings.js]: What for?

File: dev/routing/settings.js [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+    method: 'GET',
+    path: '/api/settings/{name}',
+    handler: function (request, reply) {
+      DAL.settings.getByName(request.params.name, function (err, docs) {
+        !err ? reply(docs) : reply(JSON.stringify(err));
+      });
+    }
+  });
+  server.route({
```
Comment (dmitrij-borchuk) [inline dev/routing/settings.js:16]: What for?

File: dev/publicDev/admin.menu/adminMenuDirective.jade [hunk 7]
```diff
@@ -1,5 +1,5 @@
 div.menu
-  a(href="#/properties" title="Properties") Properties
+  a(href="#/admin/properties" title="Properties") Properties
```
Comment (dmitrij-borchuk) [inline dev/publicDev/admin.menu/adminMenuDirective.jade:2]: Should be 'a(href="#/properties" title="Properties")'

File: dev/publicDev/scripts/mainCtrl.js [hunk 8]
```diff
@@ -2,10 +2,21 @@
...
+    dataService.getVisitorsCount().then(function(response){
+      vm.visitorsCount = response.data;
+    });
   }
+  ctrl.$inject = ['$http', 'dataService'];
 
-  app.controller('mainCtrl', ctrl);
+  app
+  .controller('mainCtrl', ctrl)
+  .service('dataService', dataService);
```
Comment (dmitrij-borchuk) [inline dev/publicDev/scripts/mainCtrl.js]: Use appropriate name

File: dev/publicDev/index.jade (hunk 1 above)
Comment (vetalcore) [reply dev/publicDev/index.jade:11]: resolved

File: dev/publicDev/scripts/mainCtrl.js (hunk 2 above)
Comment (vetalcore) [reply dev/publicDev/scripts/mainCtrl.js]: resolved

File: dev/routing/settings.js (hunk 6 above)
Comment (vetalcore) [reply dev/routing/settings.js:16]: resolved

File: dev/publicDev/index.jade (hunk 1 above)
Comment (vetalcore) [reply dev/publicDev/index.jade:11]: resolved

File: dev/routing.js (hunk 3 above)
Comment (vetalcore) [reply dev/routing.js]: resolved

File: dev/routing.js (hunk 4 above)
Comment (vetalcore) [reply dev/routing.js]: resolved

File: dev/publicDev/admin.menu/adminMenuDirective.jade (hunk 7 above)
Comment (vetalcore) [reply dev/publicDev/admin.menu/adminMenuDirective.jade:2]: resolved

File: dev/publicDev/scripts/mainCtrl.js (hunk 8 above)
Comment (vetalcore) [reply dev/publicDev/scripts/mainCtrl.js]: resolved

**Pull request merged**